.gitlab-ci.yml

```
@@ -538,6 +538,33 @@ test-suite:base:dev:
     expire_in: 2 week
   allow_failure: true
 
+test-suite:edge+coverage:
+  stage: build
```

Do we need 
```suggestion
  stage: build
  interruptible: true
  extends: .auto-use-tags
```
?  Or to extend something else?  Or to say @coqbot run lite ci

---

it's light not lite